### ISSUE TITLE
Stop repeating tags in many places across the module

### ISSUE THREAD

**Issue by danielhartnell** [mozilla-it/static-sites-aws#1]
I have repeatedly passed in a `tags` map to different resources in the `s3_website` module:

```
tags {
    ServiceName      = "release.mozilla.org"
    TechnicalContact = "infra-webops@mozilla.com"
    Environment      = "stage"
    Purpose          = "website"
}
```

We should write these once in our `tf_modules/s3_website/variables.tf` file and then use that variable throughout our module. Remember to set the proper variable `type`.

#### A few starter locations

https://github.com/mozilla-it/static-sites-aws/blob/master/tf_modules/s3_website/s3.tf#L30-L36
https://github.com/mozilla-it/static-sites-aws/blob/master/tf_modules/s3_website/s3.tf#L16-L22

**Comment by danielhartnell** [comment]
Additional notes:

- The `ServiceName` tag should be set based on `${var.service_name}`
- The `Environment` tag should just default to "prod" for now. We can plan to deal with other environments in the future

**Comment by danielhartnell** [comment]
Fixed in PR #4
